minecraft_servers/shortgames/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "shortgames",
+  "nice_name": "ShortGames",
+  "direct_ip": "shortgames.eu",
+  "social": {
+    "web_shop": "https://shop.shortgames.eu",
+    "discord": "https://dsc.gg/shortgames"

```

Comment:
Custom links are reserved for partners only; use a normal discord.gg/[INVITE] link instead